system/jhtree/jhinplace.cpp

```diff
@@ -1430,12 +1490,8 @@ unsigned InplaceNodeSearcher::findGE(const unsigned len, const byte * search) co
                 }
 
                 const byte * offsets = counts + numOptions * bytesPerCount;
-
-                //Avoid multiplications..  only works for 1-2 bytes per offset
-                //const byte * next = offsets + (numOptions-1) * bytesPerOffset;
-                const byte * next = offsets + ((numOptions-1) << (bytesPerOffset -1));
-
-                if (i > 0)
+                const byte * next = offsets + (numOptions-1) * bytesPerOffset;
```

Comment:
Could bytesPerOffset be 0 now ?
Previous comment suggests its only ever 1 or 2 ?

Reply:
yes.  It can now be 0 - if all the options share the same following data.  This means that whichever option is chosen, the processing will continue with the following data.